src/containers/SecretsModal/SecretsModal.js

```diff
@@ -59,15 +59,14 @@ export /* istanbul ignore next */ class SecretsModal extends Component {
       serviceAccount: '',
       annotations: [
         {
-          label: `tekton.dev/git-0`,
+          key: 'tekton.dev/git-0',
+          keyPlaceholder: 'tekton.dev/git-0',
           value: 'https://github.com',
-          placeholder: 'https://github.com',
-          id: Math.random()
-            .toString(36)
-            .substring(2, 11)
+          valuePlaceholder: 'https://github.com',
+          id: 'annotation0'
```

Comment:
as per Alan Green's [comment](https://github.com/tektoncd/dashboard/pull/188#discussion_r295946500) on using index or array length for key value: 

> key should be a value derived from the data being rendered as react uses it to determine which elements need to be updated during the render cycle in response to data change, using the array index is not a good idea as it can lead to unexpected behaviour.

Reply:
Ok... it was this before
```
Math.random()
    .toString(36)
    .substr(2, 9);
```
so I'll update the `id` to this in the SecretsModal and CreatePipelineRun modal.